lab2.js

```diff
@@ -135,12 +186,26 @@ function variablify(string) {
   // you might want to use these string methods:
   //  split(), charAt(), toUpperCase()
   // and this array method: join()
+  string = string.toLowerCase();
+  var splitString = string.split(' ');
+  var splitChar = '';
+  for (var i = 1; i < splitString.length; i++) {
+    splitChar = splitString[i].split('');
+    splitChar[0] = splitChar[0].toUpperCase();
+    splitString[i] = splitChar.join('');
+  }
+  return splitString.join('');
 }
 
 // TODO: write three more assertions
 assert(variablify('one two three') === 'oneTwoThree',
   'variablify(\'one two three\')');
-
+assert(variablify('Twenty two') === 'twentyTwo',
```

Comment:
good that you're testing your asserts with different cases - some starting with caps, others all caps, etc.